### PR TITLE
Add metric for the translation of the create argument for contract lookups

### PR DESCRIPTION
**Pull request by gerolf-da**
CHANGELOG_BEGIN
[Ledger Integration Kit] Added new metric
``daml.index.db.translate_create_argument`` to measure the duration of
the translation of the create argument from byte array to DAML LF value
that can be used by the DAML engine.
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by gerolf-da** [comment]
@SamirTalwar-DA: Thanks, I agree. But it's important to differentiate in which context the time is spent. Since `ValueSerializer` is also used when serving events, I'd rather time the use site of it with a specific timer, than just the deserialization with a generic timer. Does that make sense?

**Comment by gerolf-da** [comment]
Having said that, the name of the metric doesn't specify that it's used for active contract lookup. I'll change it to `lookup_active_contract_value_translation`.

**Comment by SamirTalwar-DA** [comment]
Sounds good. That particular deserialization method is only used in this one place, but I agree the naming is nicer at the call site.

File: ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala [hunk 1]
```diff
@@ -262,6 +262,7 @@ class Metrics(val registry: MetricRegistry) {
           registry.timer(prefix :+ "lookup_ledger_configuration")
         val lookupKey: Timer = registry.timer(prefix :+ "lookup_key")
         val lookupActiveContract: Timer = registry.timer(prefix :+ "lookup_active_contract")
+        val lookupActiveContractValueTranslation: Timer = registry.timer(prefix :+ "lookup_active_contract_value_translation")
```
Comment (stefanobaghino-da) [inline ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala:265]: Probably worth adding one level of nesting here for things that are not actually on the database.

**Comment by stefanobaghino-da** [comment]
> I don't think we're spending that much time there, but ok.

Well, glad I was wrong. :slightly_smiling_face:

**Comment by stefanobaghino-da** [comment]
Superseded by #5917

**Pull request closed without merging**